### PR TITLE
Support numpy version (<2.4 or 2.4) on CI/CD

### PR DESCRIPTION
**Pull request by kawakami-masaki0**
## Pull Request Description:
- Support numpy version (<2.4 or 2.4) on CI/CD
  - Common test
  - Pytorch test (Applied to tests for Python 3.11 and later, as numpy 2.4 is not supported in Python 3.10)
  
## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: .github/workflows/run_keras_tests.yml
Comment (yt0705) [inline .github/workflows/run_keras_tests.yml:1]: Please add a comment with the following nuance:

Since TensorFlow uses numpy<2,
it does not support verification like numpy<2.4 or numpy>=2.4 as seen in pytorch (run_pytorch_tests.yml).

File: .github/workflows/run_pytorch_tests.yml [hunk 1]
```diff
@@ -13,6 +13,16 @@ on:
 jobs:
   run-pytorch-tests:
     runs-on: ubuntu-latest
+    #  Testing was performed on combinations of Python (3.10, 3.11, 3.12), Pytorch (2.3, 2.4, 2.5, 2.6),
```
Comment (yt0705) [inline .github/workflows/run_pytorch_tests.yml]: Please add the following sentence to the comments as well.

regression testing for numpy2.4

**Review by yt0705 — CHANGES_REQUESTED** [review]
